packages/odyssey/CHANGELOG.md

```diff
@@ -37,7 +49,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Removed
 
-- Removed documentation for multiple items not yet ready for public use: Banner, Callout, Card, Dropdown, Iconography, Meter, Navigation, Switch, Toast, Top Bar, Layouts
+- [Docs] Removed documentation for multiple items not yet ready for public use: Banner, Callout, Card, Dropdown, Iconography, Meter, Navigation, Switch, Toast, Top Bar, Layouts
```

Comment:
In the future it'd be great to standardize on a convention for commits + changelog entries. [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/) were created for this reason, and we could automate the creation of the changelog once a pattern is adopted.